src/renderer/components/commands-publish-button.tsx

```diff
@@ -158,28 +159,55 @@ export class PublishButton extends React.Component<PublishButtonProps> {
     );
 
     return (
-      <fieldset disabled={isPublishing}>
-        <ButtonGroup className='button-publish'>
-            <Popover
-              content={privacyMenu}
-              position={Position.BOTTOM}
-            >
+      <>
+        <fieldset disabled={isPublishing}>
+          <ButtonGroup className='button-publish'>
+              <Popover
+                content={privacyMenu}
+                position={Position.BOTTOM}
+              >
+                <Button
+                  icon={privacyIcon}
+                />
+              </Popover>
               <Button
-                icon={privacyIcon}
+                onClick={this.handleClick}
```

Comment:
When we're in an updatable state, can we make this turn into `Update` button?

Reply:
Done 😃